sites/public/styles/blocks.scss

```diff
@@ -40,3 +40,176 @@
     }
   }
 }
+
+// .aside-block {
+//   padding: 1.25rem;
+//   border-bottom-width: 1px;
+//   border-color: var(--bloom-color-gray-400);
+// }
+
+// .listing-detail-panel {
+//   @screen md {
+//     @apply ml-16;
+//     @apply pr-2;
+//     @apply pt-0;
+//     @apply pl-0;
+//   }
+// }
```

Comment:
Can these be deleted? It looks like they are still uncommented in the shared-helpers directory